app-web/src/components/Topic/index.js

```diff
@@ -79,6 +82,9 @@ const Topic = ({ title, description, link, resources, ...rest }) => (
   <Container {...rest}>
     <TopicDecorativeBar color={TOPICS} />
     <PreviewContainer>
+      <Link to={`/editTopic/${slugify(title)}`}>
```

Comment:
can you double check to see if topics odn't already have some type of data field that has a slugified page path

Reply:
was there no slug??

Reply:
nope it's not there